hashicorp/boundary-ui
# refactor: 💡 remove usage of route-action for app route

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12615

## Description
Remove route-action usage from application route.


## How to Test
1. Use feature toggles and theme toggles to make sure functionality remained the same.

:technologist: [Admin preview](https://boundary-ui-git-icu-12615-remove-route-action-8816db-hashicorp.vercel.app/scopes/global/scopes)

## Checklist:
